frontend/src/actions/types.js

```diff
@@ -0,0 +1,16 @@
+export const SIGNUP_SUCCESS = 'SIGNUP_SUCCESS';
+export const SIGNUP_FAIL = 'SIGNUP_FAIL';
+export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
+export const LOGIN_FAIL = 'LOGIN_FAIL';
```

Comment:
I'd say remove all these extra types we don't use and add them when / if we need them

Reply:
Let's keep it. Because it's just a type not code, so it will not have conflict or any problem so far.